phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java

```diff
@@ -394,4 +402,26 @@ public ShortcutDTO getSelectedShortcut() {
     public void setSelectedShortcut(ShortcutDTO selectedShortcut) {
         this.selectedShortcut.set(selectedShortcut);
     }
+
+    public DetailsPanelAction getSelectedDetailsPanelAction() {
+        return this.selectedDetailsPanelAction.get();
+    }
+
+    public ObjectProperty<DetailsPanelAction> selectedDetailsPanelActionProperty() {
+        return this.selectedDetailsPanelAction;
+    }
+
+    public void setSelectedDetailsPanelAction(DetailsPanelAction detailsPanelAction) {
+        this.selectedDetailsPanelAction.set(detailsPanelAction);
+    }
+
+    /**
+     * Closes the currently opened details panel
+     */
+    public void closeDetailsPanel() {
+        // deselect the currently selected shortcut
+        setSelectedShortcut(null);
```

Comment:
Shouldn't we have `deselectShortcut` to be consistent?

Reply:
We can add that but internally it would still call `this.setSelectedShortcut(null)`. I'm not sure how useful such one-liner methods are...

Reply:
Ok.